File: SchoolRegister/SchoolRegister.Models/Dtos/SchoolDto.cs
using SchoolRegister.Models.Enums;

namespace SchoolRegister.Models.Dtos
{
    public class SchoolDto
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public SchoolType Type { get; set; }

        public string Settlement { get; set; }
    }
}

File: school-register-angular/src/school/dtos/school-dto.ts
export class SchoolDto {
  id: number;
  name: string;
  type: SchoolType;
  settlement: string;
}

export enum SchoolType {
  State = 1,
  Municipal = 2,
  Private = 3,
}

export const SchoolTypeEnumLocalization = {
  [SchoolType.State]: "Държавно",
  [SchoolType.Municipal]: "Общинско",
  [SchoolType.Private]: "Частно",
};
